Documentation/op-guide/performance.md

```diff
@@ -10,4 +10,4 @@ There are other sub-systems which impact the overall performance of etcd. Each s
 
 ## Benchmarks
 
-TODO
\ No newline at end of file
+To benchmark an indivisual package, etcd has Go benchmark tests. For example, mvcc package has `BenchmarkWatchableStoreWatchSyncPut` to measure how fast multiple synced watchers can receive events. To benchmark etcd performance, simply use [benchmark](https://github.com/coreos/etcd/tree/master/tools/benchmark) CLI tool. When run with 3-node cluster, of which each machine has 8 CPUs, 16GB memory, 50GB SSD, and Ubuntu 15.10 with Google Compute Engine, etcd can write around 40,000 keys per second (8-byte key, 256-byte value with 100 concurrent connections), and can do linearizable-read with the rate of 50,000 requests per second and serializable-read with the rate of 65,000 requests per second. etcd has low memory footprints, typically under 1GB. Please note that numbers vary by testing environments and workloads. We strongly encourage to run your own tests to evaluate etcd performance.
```

Comment:
let's put a spec of the machines in a different paragraph like https://github.com/coreos/etcd/blob/master/Documentation/benchmarks/etcd-2-2-0-benchmarks.md#physical-machines.

Also can we put the numbers into a table so it would be more clear?